packages/blade/src/components/BottomSheet/__tests__/__snapshots__/BottomSheet.native.test.tsx.snap

```diff
@@ -559,86 +559,104 @@ exports[`<BottomSheet /> should render Header/Footer/Body properly 1`] = `
                           }
                         >
                           <View
-                            backgroundColor="feedback.background.positive.lowContrast"
-                            platform="onMobile"
-                            size="medium"
+                            alignItems="center"
+                            data-blade-component="box"
+                            display="flex"
+                            height="28px"
+                            justifyContent="center"
                             style={
                               Array [
                                 Object {
-                                  "alignSelf": "center",
-                                  "backgroundColor": "hsla(155, 100%, 31%, 0.09)",
-                                  "borderRadius": 9999,
+                                  "alignItems": "center",
                                   "display": "flex",
-                                  "flexWrap": "nowrap",
-                                  "maxWidth": 100,
+                                  "height": 28,
```

Comment:
Why is there a fixed height now? 🤔

Reply:
So what we're doing is we have a fixed height container around icons and trailing elements and then we center align elements with respect to that fixed height.

We cannot just baseline align things because then things will be aligned by top-line of the items which looks uneven and strange.

<img width="528" alt="image" src="https://github.com/razorpay/blade/assets/30949385/9cb5944d-49e4-4dc4-a6a9-37be51d782ef">

This is similar to how Abhishek had to implement it in design as well. Title, badge, Amount, Icon all have different heights so we can't exactly top align them.